frontend/components/forms/fields/Radio/Radio.tsx

```diff
@@ -14,7 +15,7 @@ export interface IRadioProps {
   name?: string;
   className?: string;
   disabled?: boolean;
-  tooltip?: string;
```

Comment:
I don't think you need to reference `INewTooltipWrapperTipContent` here. You can simply put `tooltip?: React.ReactNode`.

I'd say if this is done other places, we can remove the reference to `INewTooltipWrapperTipContent` as I don't feel it gives us much to reference it directly.

Reply:
The type for that is now `string | JSX.Element` or `ReactNode` though, instead of just `string`. That flexibility is one of the advantages of this refactor. So this would either have to be changed in all those places from `string`, to `ReactNode`, or could just reference the parent type like it is currently so that we never have to update it again. Wdyt?

Reply:
Changed to `React.ReactNode`s per discussion